apps/damap-frontend/src/app/components/layout/layout.component.css

```diff
@@ -57,20 +100,37 @@
   background: transparent;
 }
 
-.footer-desktop {
-  font-size: small;
-  display: flex;
-  justify-content: flex-end;
-  position: sticky;
+.footer-icon {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--user-navbar-color);
+  color: var(--primary-lightest) !important;
```

Comment:
Instead of setting the color with !important, you could/should use the appropriate css class `.material-icon-white`. This is the preferred way of setting color. You could also create a new css class in `custom-theme.scss`, that sets the color and background color for icons. This way, we can reuse it, if this combination comes up in the future.

Reply:
I tried it and for me it worked, even without important. Can you remember what the circumstances were.